README.md

```diff
@@ -58,61 +58,110 @@ can see the routes with ```rake routes```;
 # Preparing a Development Environment
 
 ## Prerequisites
+---
 
  * Ruby 2.1
  * Rails 3.2.16 and the rest of the dependencies in the Gemfile
  * Postgresql 9.3 or greater (runs on at least 9.4.4)
- * A reasonable operating system (e.g., Linux)
+ * A reasonable operating system (e.g., Linux or Mac, Windows if you're saucy)
  * Various dependencies for the above
 
-After checking out the code, you'll want to install the necessary gemfiles:
+---
 
-```
-bundle install
-```
+## Setup
 
-## Database
+1. **Clone the repository to your machine.**
 
-You'll need to create a database and user:
 
-```
-$ sudo su - postgres
-$ psql
-> CREATE DATABASE bfr_webapp_db;
-> CREATE ROLE bfr_webapp_user WITH LOGIN PASSWORD 'changeme';
-> GRANT ALL ON DATABASE bfr_webapp_db TO bfr_webapp_user;
-> \q
-$ exit
-```
+  ```
+  git clone https://github.com/boulder-food-rescue/food-rescue-robot.git
+  cd food-rescue-robot
+  bundle install
+  ```
+2. **You'll need to create a database and user:**
 
-Next, copy /config/database.yml.dist to /config/database.yml and make any necessary changes.
+  ```
+  $ sudo su - postgres
+  $ psql
+  > CREATE DATABASE bfr_webapp_db;
+  > CREATE ROLE bfr_webapp_user WITH LOGIN PASSWORD 'changeme';
+  > GRANT ALL ON DATABASE bfr_webapp_db TO bfr_webapp_user;
+  > \q
+  $ exit
+  ```
 
-If you want to start with an empty schema, you can proceed as usual (rake db:setup, etc.), or you can
-load a database dump from me or elsewhere. If you start with an empty schema, you'll want to start
-by creating a Volunteer user with the admin bit set, and a first region e.g.:
+3. **Copy `database.yml.dist`**
+
+  ```
+  cp config/database.yml.dist config/database.yml
+  ```
+
+4. **Setup `config/database.yml` to look like:**
+
+  ```
+  test:
+    adapter: postgresql
+    database: bfr_webapp_db_test
+    host: localhost
+    pool: 5
+    timeout: 5000
+
+  development:
+    adapter: postgresql
+    database: bfr_webapp_db
+    host: localhost
+    pool: 5
+    timeout: 5000
+
+  production:
+    adapter: postgresql
+    database: bfr_webapp_db
+    username: bfr_webapp_user
+    password: CHANGEME
+    host: localhost
+    pool: 5
+    timeout: 5000
+  ```
+  **Warning:** _The database defined as "test" will be erased and re-generated from your  development database when you run "rake". Do not set this db to the same as development or  production._
+
+5. **In a new terminal tab/window:**
+
+  ```
+  bundle exec rake db:create
+  ```
+
+6. **If you receive an encoding error, go to db console and:**
+
+  ```
+  SET CLIENT_ENCODING TO 'WIN1252';
```

Comment:
Is this a windows-specific issue? Do we really need to mention this? I've yet to come across someone running this on windows, but wouldn't utf-8 work ok?